src/main/java/javaslang/collection/List.java

```diff
@@ -1348,6 +1354,32 @@ public T head() {
         }
 
         @Override
+        public Tuple2<List<T>, List<T>> splitAt(Predicate<? super T> predicate) {
+            Tuple2<List<T>, List<T>> t = splitByPredicateFirstRound(this, predicate);
+            return Tuple.of(t._1.reverse(), t._2);
+        }
+
+        @Override
+        public Tuple2<List<T>, List<T>> splitAtInclusive(Predicate<? super T> predicate) {
+            Tuple2<List<T>, List<T>> t = splitByPredicateFirstRound(this, predicate);
+            if (!t._2.isEmpty()) {
```

Comment:
(minor) I would test the positive condition `isEmpty()` and swap the cases. It is less information for the human brain to process and `!` can be overseen